code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -0,0 +1,42 @@
+/* global React, dashboard */
+
+window.dashboard = window.dashboard || {};
+
+/**
+ * Stage progress component used in level header and course overview.
+ */
+window.dashboard.CourseProgress = (function (React) {
```

Comment:
Do they need to reference this component? afaict, the only reference currently is progress.renderCourseProgress...which I'm realizing now is still in app/assets rather than code-studio :(